PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackV0Extended.cxx

```diff
@@ -268,6 +270,21 @@ struct femtoUniversePairTaskTrackV0Extended {
       pairCloseRejection.init(&resultRegistry, &qaRegistry, ConfCPRdeltaPhiCutMin.value, ConfCPRdeltaPhiCutMax.value, ConfCPRdeltaEtaCutMin.value, ConfCPRdeltaEtaCutMax.value, ConfCPRChosenRadii.value, ConfCPRPlotPerRadii.value);
       pairCloseRejectionV0.init(&resultRegistry, &qaRegistry, ConfCPRdeltaPhiCutMin.value, ConfCPRdeltaPhiCutMax.value, ConfCPRdeltaEtaCutMin.value, ConfCPRdeltaEtaCutMax.value, ConfCPRChosenRadii.value, ConfCPRPlotPerRadii.value);
     }
+
+    if (!ConfLocalEfficiency.value.empty()) {
+      TFile* pf = TFile::Open(ConfLocalEfficiency.value.c_str(), "read");
```

Comment:
would it be possible to avoid raw pointer also here?

Reply:
It should be automatically convertible to std::unique_ptr or std::shared_ptr.

Reply:
Done. Thanks for the suggestion.